public/js/customer/deposit/customer_deposit.js

```diff
@@ -25,51 +30,111 @@ var tagify_d_bank_account = new Tagify(elm_d_bank_account, {
     //         "email": "jhattersley0@ucsd.edu"
     //     },
     // ]
-})
+});
+
+tagify_d_bank_account.on("dropdown:select", onBankSelectSuggestion);
+tagify_d_bank_account.on("input", onBankInput);
 
-tagify_d_bank_account.on('dropdown:show dropdown:updated', onDropdownShow)
-// tagify_d_bank_account.on('dropdown:select', onReceiptCustomerSelectSuggestion)
-tagify_d_bank_account.on('input', onReceiptCustomerInput)
-// tagify_d_bank_account.on('remove', onReceiptCustomerRemove)
+function createRecordsToDeposit(f) {
+    console.log(f);
+    // counter for ids
 
-var addAllSuggestionsElm;
+    counter++;
 
-function onDropdownShow(e){
-    // var dropdownContentElm = e.detail.tagify_d_bank_account.DOM.dropdown.content;
+    // tr template
+    let inner = `
+     <tr data-id="${counter}" id="b_item_entry_${counter}">
+        <td class="table-item-content">${f.date}</td>
+        <td class="table-item-content">${f.customer_name}</td>
+        <td class="table-item-content">${f.payment_method}</td>
+        <td class="table-item-content">${f.value}</td>
+        <td class="table-item-content">${f.total_amount_received}</td>
+        <td class="table-item-content">
+            <div class="form-check">
+                <input type="checkbox" class="form-check-input" onchange="handleCheck(this,'${f.payment_method}',${f.total_amount_received})" id="${f.value}" name="is_deposited[]" value="${f.value}">
+            </div>
+        </td>
+    </tr>
+     `;
+    // append to table
+    $("#deposit-list").append(inner);
 }
 
-function onReceiptCustomerSelectSuggestion(e){
-    // checks for data of selected customer
-    // console.log(e.detail.data);
+function onBankSelectSuggestion() {
+    $("#deposit-list").empty();
+    // Get data from server.
+    var request = $.ajax({
+        url: "/ajax/get/receipts",
+        method: "GET",
+    });
 
-    // $("#r_customer_id").val(e.detail.data.value)
-    // $("#r_tin_number").val(e.detail.data.tin_number)
-    // $("#r_contact_person").val(e.detail.data.contact_person)
-    // $("#r_mobile_number").val(e.detail.data.mobile_number)
-}
+    request.done(function (res, status, jqXHR) {
+        console.log(res);
+        for (i = 0; i < res.length; i++) {
+            createRecordsToDeposit(res[i]);
+        }
+    });
 
-function onReceiptCustomerRemove(e){
-    // $("#r_customer_id").val("")
-    // $("#r_tin_number").val("")
-    // $("#r_contact_person").val("")
-    // $("#r_mobile_number").val("")
+    request.fail(function (jqXHR, status, error) {
+        console.log(error);
+    });
```

Comment:
Kindly leave this function blank. Re-selecting banks should not reload contents.